_opcodes/op017-bg2.html

```diff
@@ -25,6 +25,6 @@
 
 <br/><br/>The upper word of the 'Type' field can further control the style.
 <br/>Known values for the upper word of 'Type' are:
-<br/>1 Cure death
-<br/>2 Removes all non timing-mode-9 effects
-<br/>3 Cure death and remove non timing-mode-9 effects
+<br/>BIT0 – Raise dead
+<br/>BIT1 – Remove all effects with a limited timing mode (not equipped(2) or permanent(9)), but also any ITM in the Magical Weapon slot.
+<br/>BIT2 – Raise dead and remove all effects with a limited timing mode (not equipped(2) or permanent(9)), but also any ITM in the Magical Weapon slot.
```

Comment:
This is introducing an incorrect statement. If you think enumerating them as bits is better than what's currently there, then BIT2 should go away completely. It has no known function.

Reply:
You're right, fixed.